app/concierge/mycore/templates/TenantForm.html

```diff
@@ -0,0 +1,20 @@
+{% extends "index.html" %}
+{% block content %}
+    <p><h3>Add New Tenant</h3>
+            <form  method="post" enctype="multipart/form-data">
+                {% csrf_token %}
+                <p><label>Enter Tenant First Name</label>
```

Comment:
maybe you have to use `form.as_p`(something like this) instead of "hardcoding" these values

Reply:
Thanks for idea))